README.md

```diff
@@ -417,6 +417,42 @@ Note that this configuration can be customized in terms of where Ollama is confi
 
 Please refer to the [Ollama documentation for further details](https://github.com/ollama/ollama/tree/main/docs).
 
+### Building sysext images
+
+To use the build scripts in this repository, the following packages are required:
+
+- `curl`
+- `jq`
+- `squashfs-tools`
+- `xz-utils`
+- `gawk`
+- [`yq`](https://github.com/mikefarah/yq/releases/latest/)
+
+
+#### Build individual sysext image
+
+To build the Kubernetes sysext for example, use:
+
+```sh
+./create_kubernetes_sysext.sh v1.29.8 kubernetes
+```
+
+Afterwards, you can test the sysext image with:
+
+```sh
+sudo cp kubernetes /etc/extensions/kubernetes.raw
```

Comment:
I think it is like this
`sudo cp kubernetes.raw /etc/extensions/kubernetes.raw`